db/migrate/20210911182500_add_columns_for_case_development_analytics.rb

```diff
@@ -0,0 +1,15 @@
+class AddColumnsForCaseDevelopmentAnalytics < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :monitoree_snapshots, :isolation_to_exposure, :integer
+    remove_column :monitoree_snapshots, :exposure_to_isolation, :integer
+
+    add_column :monitoree_snapshots, :exposure_to_isolation_active, :integer
+    add_column :monitoree_snapshots, :exposure_to_isolation_not_active, :integer
+    add_column :monitoree_snapshots, :cases_closed_in_exposure, :integer
+    add_column :monitoree_snapshots, :isolation_to_exposure, :integer
+
+    add_column :patients, :enrolled_workflow, :string
```

Comment:
I'm fine with this change... @tstrass thoughts?

Reply:
hmm, since we're now going with a gobal limit of 200 instead and a migration would be required if we needed to add more workflows, I would lean more towards making this a binary field because it's more consistent with the `isolation` field which will make it easier for others to understand

Reply:
Fair enough, I'm good with switching it to a bool

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1085/commits/7ab0b51129f946962aa6642833fbed4e63ce873b

Reply:
I think you missed a couple places: https://github.com/SaraAlert/SaraAlert/pull/1085/files#diff-3346374057035485f7d2ab03927877c081ce3af8eff7a01994fe76211256d9ebR269

Reply:
No that's declaring an `enrolled_workflow` variable for exports so that Isolation or Exposure text is displayed

Reply:
ok, this isn't necessary then: https://github.com/SaraAlert/SaraAlert/pull/1085/files#diff-850172f7658c1cabc6010b0d7e23c668251b024c7aad38651260e458ecee9478R134

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/1085/commits/55ee9184221cc82f340595157ed9580072b9df47